lib/components/Footer/Footer.tsx

```diff
@@ -27,6 +27,7 @@ export const Footer = forwardRef<HTMLDivElement, FooterProps>(function Footer(
   return (
     <footer
       ref={ref}
+      role="contentinfo"
```

Comment:
Based on the [mdn](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/contentinfo_role) `contentInfo` is already present when using correct semantic, `<footer>`.

But it might not hurt to be explicit.